README.md

```diff
@@ -351,6 +351,7 @@ where:
 
 * `suggestion` - the selected suggestion
 * `suggestionValue` - the value of the selected suggestion (equivalent to `getSuggestionValue(suggestion)`)
+* `suggestionIndex` - the index of the selected suggestion in the `suggestions` array )
```

Comment:
The `)` at the end should probably not be there. Also, `suggestionIndex` need to be added to the signature few lines above.